include/ck/tensor_operation/gpu/element/unary_element_wise_operation.hpp

```diff
@@ -25,17 +68,24 @@ struct PassThroughPack2
     constexpr const static bool is_pack2_invocable = true;
 };
 
-struct PassThrough
+struct PassThrough : public UnaryOpBase
```

Comment:
I am suspicious about moving element ops from compile time to run time... Could you run some benchmarks to make sure we are not losing performance for all our element ops?

Reply:
Generally I think constexpr keyword here is nonsense because you can't perform Relu operations etc. during compilation (the exception would be operations for Number<> but we don't have such reloads anyway), I'll try to add some measurements.

Reply:
Ok, I restored constexpr keyword to be 100% safe. I also measured scaleaddscaleadd from convnd_active example and perforamnce looks the same like before change. Thanks for catch.